README.md

```diff
@@ -1,51 +1,83 @@
+
+Hence, the video gestalts can be read in two ways: 1- an overall quick glance shows all the scenes of the entire video, 2- by focusing on one animated thumbnail, we can watch the entire video, by starting in the upper left corner, and following to the right, then descending one block lower and moving from right to left, then descending one block and moving left to right again, etc.
+
+A longer explanation is available in [this blog post](https://eamonn.org/video-gestalt-one-glance-overview-of-a-video).
+
 ## Installation
 
-So far this has only been tested on Linux and Chrome OS, but it will likely work on MacOS too, and maybe even on Windows.
+So far this has been tested on Linux, Chrome OS and Windows, but it will likely work on MacOS too.
 
-The only file you need from this repo is `gestalt.py`. You can grab this however you want, and make it executable. For example, do the following from the command line:
+To install, simply use `pip`:
 
 ```bash
-wget https://raw.githubusercontent.com/eobrain/videogestalt/main/gestalt.py
-chmod +x gestalt.py
+pip install --upgrade videogestalt
 ```
 
-If they are not already installed, you will need to install `python3` and the corresponding Python package manager `pip`.
-
+If they are not already installed, you will need to install `python3` and the corresponding Python package manager `pip` beforehand.
 
 On Linux and friends you might be able to do this like so:
 ```bash
 sudo apt-get install python3 python3-pip
 ```
 
-You will need to install the `moviepy` Python library:
+## Usage
+
+An executable binary `videogestalt` is automatically installed in the local environment.
+
+The following examples assume you cloned the repository to get access to the `example` folder.
+
+To generate a video file:
+
+```bash
+videogestalt -i example/test.mp4 -o test-gestalt --video
+```
+
+To generate an animated GIF (warning, output can be large):
 
 ```bash
-pip install moviepy
+videogestalt -i example/test.mp4 -o test-gestalt --gif
 ```
 
+The application can also be used as a Python module:
 
-## Usage
+```python
+>>> from videogestalt import videogestalt as vg
+>>> vg.main('-i example/test.mp4 -o test-gestalt --gif')
+```
 
-Put the `gestalt.py` in the same directory as an input video file `test.mp4`.
+## Building
 
-Generate a video file:
+The module can be built with PEP517 standard tools, such as `pypa/build`:
 
 ```bash
-./gestalt.py -i test.mp4 -v
+python -sBm build .
 ```
 
-Generate an animated GIF (warning, can be large):
+It can also be installed in development/editable mode after cloning this git repository:
 
 ```bash
-./gestalt.py -i test.mp4 -g
+pip install --upgrade -e .
```

Comment:
For me (using Python 3.9.2 on Linux) this failed with

```
ERROR: File "setup.py" not found. Directory cannot be installed in editable mode: /home/eobrain/src/lrq3000
(A "pyproject.toml" file was found, but editable mode currently requires a setup.py based build.)
```

Reply:
Mmmm I guess your pip version is not updated to the latest, as it should be fixed since pip versions of [last year](https://github.com/pypa/setuptools/issues/2816). Nevertheless, I have implemented a workaround now, please retry with my updated PR and let me know if this works :-)